bulbs/ads/static/js/mobile-ad-placer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var _ = require('lodash/lodash');
+var adHtml = '<div class="dfp dfp-slot-inread" data-ad-unit="inread"></div>';
+
+
+var MobileAdPlacer = {
+  placeAds: function () {
+    var body = $('body');
+    var paragraphs = body.find('p');
+    var wordCount = 0;
+    var adsPlaced = 0;
+
+    for (var i = 0; i < paragraphs.length; i++) {
```

Comment:
it does, but my chump self already released this. I don't think it's worth going through the bulbs deploy process again just to change that out. :-( X10.